_inc/client/main.jsx

```diff
@@ -267,6 +268,19 @@ class Main extends React.Component {
 	}
 
 	render() {
+		if ( this.props.route.path === '/plans-prompt' ) {
```

Comment:
Instead of creating a completely new return just for that new prompt, what do you think about moving that logic inside the existing `renderMainContent()`, so we can keep `render` as simple as possible?

Reply:
One of the requirements was that the plans prompt doesn't show anything else than the intro banner and the plans grid. That's why we're returning early with a different set of components. @jeherve is there a cleaner/better way to do it?

Reply:
You should be able to control what other things appear with things like `this.shouldShowSupportCard()`, for example.

Reply:
Sounds good. This has been addressed in https://github.com/Automattic/jetpack/pull/13348/commits/0a37a0123fec908936cd640119fa003945c9a2ce.